pkg/config/validation/validate_test.go

```diff
@@ -0,0 +1,122 @@
+package config
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
+
+	"cuelang.org/go/cue/load"
+)
+
+const schemaPath = "../schema"
+
+func TestValidateConfig(t *testing.T) {
+	var table = []struct {
+		description string
+		filename    string
+		kind        string
+		hasError    bool
+		errString   string
+	}{
+		{
+			description: "valid bundle config",
+			filename:    "./testdata/valid/bundle.json",
+			kind:        olmBundle,
+			hasError:    false,
+		},
+		{
+			description: "valid package config",
+			filename:    "./testdata/valid/package.json",
+			kind:        olmPackage,
+			hasError:    false,
+		},
+		{
+			description: "invalid bundle config",
+			filename:    "./testdata/invalid/bundle.json",
+			kind:        olmBundle,
+			hasError:    true,
+			errString:   `#olmbundle.relatedImages.0.name: incomplete value !=""`,
+		},
+		{
+			description: "invalid channel property config",
+			filename:    "./testdata/invalid/channel_property.json",
+			kind:        olmChannel,
+			hasError:    true,
+			errString:   `#olmchannel.value.name: conflicting values !="" and 1 (mismatched types string and int)`,
+		},
```

Comment:
this makes me wonder if you can the validator will pass any valid top-level channel_property